File: 백준/Bronze/10818. 최소， 최대/최소， 최대.js
const fs = require('fs');
const file = process.platform === "linux" ? "/dev/stdin" : "input.txt";
const input = fs.readFileSync(file).toString().split("\n");

const n = Number(input[0]);
const arr = input[1].split(" ").map(Number);
const min = Math.min(...arr);
const max = Math.max(...arr);
console.log(min, max);

File: 백준/Bronze/10818. 최소， 최대/최소， 최대.py
N = int(input())
arr = list(map(int, input().split()))

min_v = arr[0]
max_v = arr[0]

for i in range(1, N):
    if min_v > arr[i]:
        min_v = arr[i]
    if max_v < arr[i]:
        max_v = arr[i]

print(f'{min_v} {max_v}')